src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -41,14 +41,14 @@
     private transient final KubernetesCloud cloud;
 
     @DataBoundConstructor
-    public KubernetesSlave(PodTemplate template, String nodeDescription, KubernetesCloud cloud, Label label)
+    public KubernetesSlave(PodTemplate template, String nodeDescription, KubernetesCloud cloud, String labelStr)
```

Comment:
we need to keep backwards compatibility, can you deprecate the old method and add the new one?